Rubberduck.Core/Navigation/CodeExplorer/CodeExplorerCustomFolderViewModel.cs

```diff
@@ -19,11 +20,14 @@ public class CodeExplorerCustomFolderViewModel : CodeExplorerItemViewModel
         };
 
         private readonly IProjectsProvider _projectsProvider;
+        private readonly IVBE _vbe;
```

Comment:
Given `IVBE` being among the very last objects that get destroyed (if at all - or did that change?), does this field not make the viewmodel live longer than it needs to?

Reply:
This is provided by CW, which is the same instance that we got from the `Extension.cs` class. A separate PR would need to review all the VBE usage and ensure that the classes are dead by time the .NET parts shut down.